pilot/info/queuedata.py

```diff
@@ -139,7 +140,14 @@ def clean(self):
                 self.container_options = self.container_options.replace(" --contain", ",${workdir} --contain")
                 logger.info("Note: added missing ${workdir} to container_options/singularity_options: %s" % self.container_options)
 
-        pass
+        # set default copytools for es_events and es_events_read
```

Comment:
I would rather pass `objectstore` as default_copytools in Stage Client instance.
 https://github.com/PanDAWMS/pilot2/blob/next/pilot/api/data.py#L42